### PR TITLE
Wip/hughsie/ccgx replug

### PR DESCRIPTION
**Pull request by hughsie**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [X] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

File: plugins/ccgx-dmc/fu-ccgx-dmc-device.c [hunk 1]
```diff
@@ -690,39 +692,39 @@ fu_ccgx_dmc_device_attach(FuDevice *device, FuProgress *progress, GError **error
...
 
+	/* the user has to do something */
+	if (manual_replug) {
+		g_autoptr(FwupdRequest) request = fwupd_request_new();
+		fwupd_request_set_kind(request, FWUPD_REQUEST_KIND_IMMEDIATE);
+		fwupd_request_set_id(request, FWUPD_REQUEST_ID_REMOVE_REPLUG);
+		fwupd_request_add_flag(request, FWUPD_REQUEST_FLAG_ALLOW_GENERIC_MESSAGE);
+		fwupd_request_set_message(
+		    request,
+		    "The update will continue when the device USB cable has been "
```
Comment (superm1) [inline plugins/ccgx-dmc/fu-ccgx-dmc-device.c:720]: We've got this string in like 3 or 4 plugins now, can we make it generic?

Comment (hughsie) [reply plugins/ccgx-dmc/fu-ccgx-dmc-device.c:720]: It literally is `FWUPD_REQUEST_FLAG_ALLOW_GENERIC_MESSAGE` :) Or do you mean "if we set FWUPD_REQUEST_ID_REMOVE_REPLUG then automatically use this text"?

Comment (superm1) [reply plugins/ccgx-dmc/fu-ccgx-dmc-device.c:720]: I mean the string literally. Like can we have a define for it?

Comment (hughsie) [reply plugins/ccgx-dmc/fu-ccgx-dmc-device.c]: Ahh, I think we can do better than that -- how about the follow-up commit?

**Review by superm1 — APPROVED** [review]

**Pull request merged**